modules/local/deepvariant/main.nf

```diff
@@ -0,0 +1,46 @@
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process DEEPVARIANT {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::deepvariant=1.2.0" : null)
```

Comment:
Iirc, deepvariant on bioconda doesn't contain the run_deepvariant wrapper script. I suppose this won't affect us since we'll be using singularity containers, but I am not sure if its better to comment this line out :)